lib/instance-functions

```diff
@@ -3,8 +3,11 @@
 # instance-functions
 #
 # List, run, start, stop and ssh to Amazon AWS EC2 instances
-
-source $(dirname ${BASH_SOURCE[0]})/shared.inc
+if [[ ${SHELL} = "/bin/zsh" ]]; then
```

Comment:
This is not going to work ok Debian and it's derivatives where zsh is in `/usr/bin/zsh`